src/containers/ReviewContainer.jsx

```diff
@@ -25,26 +25,32 @@ type Props = {
     getTxFee: boolean
   },
   error: any,
-  userProfile: Object ,
+  userProfile: Object,
   recipients: Array<Recipient>,
-  push: Function,
+  push: Function
 }
 
 class ReviewContainer extends Component<Props> {
   render () {
     const { cryptoPrice, txFee, transferForm, currency } = this.props
-    const toCurrencyAmount = cryptoAmount =>
-      utils.toCurrencyAmount(
-        cryptoAmount,
-        cryptoPrice[JSON.parse(transferForm.accountId).cryptoType],
-        currency
-      )
+    const toCurrencyAmount = (cryptoAmount, txFee) =>
```

Comment:
suggest param "usePlatformType: Boolean" instead of txFee for better readability.
Also add flow type info and comment